compose/release-notes.md

```diff
@@ -8,6 +8,38 @@ redirect_from:
 ---
 {% include compose-eol.md %}
 
+## 2.18.0
+{% include release-date.html date="2023-05-16" %}
+
+### Enhancements
```

Comment:
Let's combine `bug fixes` and `enhancements` to keep it consistent with previous release notes